widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetContract.kt

```diff
@@ -4,6 +4,8 @@ import android.app.Activity
 import com.glia.widgets.base.BaseController
 import com.glia.widgets.base.BaseView
 
+private const val NO_ORDER = Int.MIN_VALUE
```

Comment:
MIN_VALUE will actually show an item at the very top. So, it like the highest order in Entry Widget. Maybe it makes sense to write `private const val LOWEST_ORDER = Int.MAX_VALUE` and use `LOWEST_ORDER` for `ProvidedBy` as well?

Reply:
I know it will not happen, but in the case the `provided by` and the `error` items meet, the error item should have higher priority, so the order is right, but the naming doesn't reflect this. so maybe it makes sense to rename the `NO_ORDER` to `HIGHER_ORDER` or `DEFAULT_ORDER` or just put `Int.MIN_VALUE`?

Reply:
changed to `private const val HIGHEST_ORDER = Int.MIN_VALUE`